src/angulartics-gosquared.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * @license Angulartics v0.17.2
+ * (c) 2013 Luis Farzati http://luisfarzati.github.io/angulartics
+ * Universal Analytics update contributed by http://github.com/willmcclellan
+ * License: MIT
+ */
+(function(angular) {
+'use strict';
+
+/**
+ * @ngdoc overview
+ * @name angulartics.gosquared
+ * Enables analytics support for GoSquared (https://www.gosquared.com)
+ */
+angular.module('angulartics.gosquared', ['angulartics'])
+.config(['$analyticsProvider', function ($analyticsProvider) {
+
+  $analyticsProvider.settings.trackRelativePath = true;
+
+  // Set the default settings for this module
+  $analyticsProvider.settings.gosquared = {
+    // Whether to send person details along with events.
+    sendPersonDetailsWithEvents: false
+  };
+
+  $analyticsProvider.registerPageTrack(function (path) {
+    if (window._gs) {
```

Comment:
You should use `waitForVendorApi` instead, see https://github.com/luisfarzati/angulartics#creating-your-own-vendor-plugin